postgres/lib.w

```diff
@@ -66,7 +66,8 @@ class DatabaseSim impl IDatabase {
       env: {
         POSTGRES_PASSWORD: "password"
       },
-      containerPort:5432
+      containerPort: 5432,
+      volumes: ["$WING_STATE_DIR:/var/lib/postgresql/data"],
```

Comment:
Maybe add a platform parameters to control the default

Reply:
Would it help if the Wing Console had a way to reset an app's state?

Reply:
Feel free to open a ticket and I can add a platform parameter for the postgres lib if you think there are use cases